e2e/paths/uploadDocuments_test.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../config.js');
+
+Feature('UploadDocuments');
+
+Before((I, caseViewPage) => {
+	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+	caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+});
+
+Scenario('Selecting to follow for a document produces a textarea', (I, uploadDocumentsPage) => {
+	I.selectOption(uploadDocumentsPage.fields.socialWorkChronologyStatus, 'To follow');
```

Comment:
Having page object responsibility split (as described above) definitely helps here so we could just call one or two methods from page objects instead of repeating code.

Reply:
How granular do you want me to go? I don't want to have 8 methods for uploading files and filling in forms....

 I could maybe make two generic functions actually and pass in params? Thoughts?